doc/src/input/md.md

```diff
@@ -169,16 +169,17 @@ thermostat = {type = "Rescale", temperature = "250 K", tolerance = "10 K"}
 
 Control algorithm are supplementary steps that modify the system to ensure some
 invariant, or apply some constraint. They are specified in the `controls` array,
-by giving a control `type`.
+by giving a control `type`. The `every` key specifies that the algorithm should
+only be run every `n` step of the simulation (optional, defaults to 1).
 
 ```toml
 [simulations.propagator]
 type = "MolecularDynamics"
 timestep = "1 fs"
 controls = [
-    {type = "RemoveRotation"}
+    {type = "RemoveRotation", every = 4}
```

Comment:
Sure ! Maybe writing `every 4 timestep` would be clearer ?I feel that `every 4 fs` could lead one to believe they are expected to set a time in fs rather than a number of timesteps.